docs/can-guides/commitment/migrating_to_3.0.md

```diff
@@ -225,6 +237,146 @@ If you were using `can.view.preload` then use [can-stache.registerPartial] inste
 stache.registerPartial("some-id", renderer);
 ```
 
+### API changes
+
+A number of API changes have been made in CanJS 3.
+
+#### can.event
+
+Some methods have been renamed in `can.event`.
+
+```js
+can.event.addEvent(el, 'click', function() {});
+can.event.removeEvent(el, 'click', function() {});
+```
+
+Becomes:
+
+```js
+can.event.addEventListener(el, 'click', function() {});
```

Comment:
`addEventListener` doesn't take the element as the first parameter, so this needs to be `can.event.addEventListener.call(el, 'click', function() {});`.